scripts/addReward.js

```diff
@@ -0,0 +1,73 @@
+// ES5 style
+const config = require("./deploymentConfig");
+const thorify = require("thorify").thorify;
+const Web3 = require("web3");
+const Multirewards = require(config.pathToMultirewardsJson);
+const assert = require('assert');
+
+let network = null;
+let multirewardsAddress = null;
+let duration = null;
+
+if (process.argv.length < 5) 
+{
+    console.error("Usage: node scripts/addReward [mainnet|testnet] [Multirewards address] [Duration in days]");
+    process.exit(1);
+} 
+else
+{
+    if (process.argv[2] == "mainnet") network = config.network.mainnet;
+    else if (process.argv[2] == "testnet") network = config.network.testnet;
+    else {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
```

Comment:
Could this be written as:
```typescript
network = config.network[process.argv[2];
if (network === undefined) { console.error }
```

Reply:
I know this is fairly standard code from other deploy scripts too; but on reading it just now after seeing Oli's comment, I am also wondering if we also might want to add some additional ceremony around the actual production deployment? e.g. like requiring a confirmation from the user, if mainnet is requested.

I am just thinking ahead to building more standard patterns/processes around our blockchain projects. We might end up using other tools rather than writing our own, making this irrelevant, but it could be useful practice to just keep adding more rigour where we see it.

(Also if any change is made here, obviously the same would apply to the other scripts)

Reply:
Yeah I thought about it before, I think it's a good thing to have more confirmation esp on mainnet. Let me add this to all the relevant repos we have.